src/reporting/quicksight-custom-resource.ts

```diff
@@ -104,62 +104,817 @@ export function createQuicksightCustomResource(
     templateId: props.templateId,
     dataSourceArn: props.dataSourceArn,
     databaseName: databaseName,
-    dataSets: [
-      //Base View
+    dataSets: _getDataSetDefs('no', eventViewColumns, eventViewProjectedColumns, tenYearsAgo, futureDate),
+    dataSetsSpice: _getDataSetDefs('yes', eventViewColumns, eventViewProjectedColumns, tenYearsAgo, futureDate),
+  };
+
+  console.log('props.useSpice:', props.useSpice);
```

Comment:
use logger.debug()

Reply:
It's fine to use `info` level logging for custom resource Lambda.